.github/workflows/promote.yaml

```diff
@@ -61,11 +61,12 @@ jobs:
           npx lerna exec --ignore @nuxeo/nuxeo-elements-storybook -- npm publish --@nuxeo:registry=https://packages.nuxeo.com/repository/npm-public/
 
       - name: Publish to https://registry.npmjs.org
+        continue-on-error: true
         if: ${{ !github.event.inputs.dryRun }}
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_REGISTRY_TOKEN }}
         run: |
-          npx lerna exec --ignore @nuxeo/nuxeo-elements-storybook -- npm publish --@nuxeo:registry=https://registry.npmjs.org --access public
+          npx lerna exec --ignore @nuxeo/nuxeo-elements-storybook -- npm publish --@nuxeo:registry=https://registry.npmjs.org/ --access public
```

Comment:
🤯 we were missing the `/` and that prevented us from publishing to the registry?

Reply:
I am not sure. I did not test it, but I think we have faced this in the past. I appended the `/` for our next run, to see if it helps. :P

Reply:
😂 well, it won't hurt to do it next time ✨

Reply:
🤞